suite-native/state/src/extraDependencies.ts

```diff
@@ -1,25 +1,54 @@
+import { Platform } from 'react-native';
+
 import { ExtraDependencies } from '@suite-common/redux-utils';
 import { extraDependenciesMock } from '@suite-common/test-utils';
 import { supportedNetworkSymbols } from '@suite-native/config';
 import { selectDevices } from '@suite-common/wallet-core';
 import { selectFiatCurrencyCode, setFiatCurrency } from '@suite-native/module-settings';
 import { PROTO } from '@trezor/connect';
 import { mergeDeepObject } from '@trezor/utils';
+import { NativeUsbTransport } from '@trezor/transport-native';
+
+const protobugMessages = require('@trezor/protobuf/messages.json');
+
+const transports = Platform.select({
+    ios: ['UdpTransport'],
+    android: [
+        new NativeUsbTransport({
+            messages: protobugMessages,
```

Comment:
nic typo-bug `probug` :D. I would prefer assigning `'@trezor/protobuf/messages.json'` as default inside `AbstractTransport` in transport package. But as long this works too, no problem with it as it is. 

tldr - we might need in the future to support passing custom messages to transport (because of some shitcoin-plugins-modularization), but this would be happening on connect level anyway.

Reply:
I will keep it here now since I don't want to complicate things in this PR.

typo fixed :D